rpw/db/plumbing.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf8
+
+"""
+Plumbing Wrapper
+"""
+
+from rpw.db import Element
+from rpw import revit, DB
+
+
+class FluidType(Element):
+    """
+    Autodesk.Revit.DB.Plumbing.FluidType wrapper Based on rpw.db.Element
+    >>> from rpw.db import FluidType
+
+    Or a dictionnary of fluids used by system
```

Comment:
I generally add 1 or 2 methods as an example of how the class is used, but 
details about the method go in the actual method definition below. 
For example:
https://github.com/gtalarico/revitpythonwrapper/blob/master/rpw/db/spatial_element.py#L81